documentation/announcing-az-module.md

```diff
@@ -0,0 +1,163 @@
+| AzureRM.Backup | Az.Backup***|
+| AzureRM.Batch | Az.Batch |
+| AzureRM.Billing | Az.Billing |
+| AzureRM.Cdn | Az.Cdn |
+| AzureRM.CognitiveServices | Az.CognitiveServices |
+| AzureRM.Compute | Az.Compute |
+| AzureRM.Compute.ManagedService | Az.Compute.ManagedService*** |
+| AzureRM.Consumption | Az.Consumption* |
+| AzureRM.ContainerInstance | Az.ContainerInstance* |
+| AzureRM.ContainerRegistry | Az.ContainerRegistry* |
+| AzureRM.DataFactories | Az.DataFactories*** |
+| AzureRM.DataFactoryV2 | Az.FataFactoryV2* |
+| AzureRM.DataLakeAnalytics | Az.DataLakeAnalytics |
+| AzureRM.DataLakeStore | Az.DataLakeStore |
+| AzureRM.DataMigration | Az.DataMigration |
+| AzureRM.DeviceProvisioningServices | Az.DeviceProvisioningServices* |
+| AzureRM.DevSpaces | Az.DevSpaces |
+| AzureRM.DevTestLabs | Az.DevTestLabs |
+| AzureRM.Dns | Az.Dns |
+| AzureRM.EventGrid | Az.EventGrid |
+| AzureRM.EventHub | Az.EventHub |
+| AzureRM.HDInsight | Az.HDInsight** |
+| AzureRM.Insights | Az.Insights* |
+| AzureRM.IotHub | Az.IoTHub |
+| AzureRM.KeyVault | Az.KeyVault|
+| AzureRM.LogicApp | Az.LogicApp*** |
+| AzureRM.MachineLearning | Az.MachineLearning |
+| AzureRM.MachineLearningCompute | Az.MachineLearningCompute* |
+| AzureRM.ManagedServiceIdentity | Az.ManagedServiceIdentity* |
+| AzureRM.ManagementPartner | Az.ManagementPartner* |
+| AzureRM.Maps | Az.Maps |
+| AzureRM.MarketplaceOrdering | Az.MarketplaceOrdering* |
+| AzureRM.Media | Az.Media |
+| AzureRM.Network | Az.Network* |
+| AzureRM.NotificationHubs | Az.NotificationHubs* |
+| AzureRM.OperationalInsights | Az.OperationalIsights |
+| AzureRM.PolicyInsights | Az.PolicyInsights *|
+| AzureRM.PowerBIEmbedded | Az.PowerBIEmbedded* |
+| AzureRM.Profile | Az.Profile * |
+| AzureRM.RecoveryServices | Az.RecoveryServices**|
+| AzureRM.RecoveryServices.Backup | Az.RecoveryServices**|
+| AzureRM.RecoveryServices.SiteRecovery | Az.RecoveryServices**|
+| AzureRM.RedisCache | Az.RedisCache** |
+| AzureRM.Relay | Az.Relay* |
+| AzureRM.Reservations | Az.Reservations* |
+| AzureRM.Resources | Az. Resources |
+| AzureRM.Scheduler | Az.Scheduler*** |
+| AzureRM.Search | Az.Search |
+| AzureRM.Security | Az.Security* |
+| AzureRM.ServiceBus | Az.ServiceBus |
+| AzureRM.ServiceFabric | Az.ServiceFabric** |
+| AzureRM.SignalR | Az.SignalR |
+| AzureRM.Sql | Az.Sql |
+| AzureRM.StorageSync | Az.StorageSync |
+| AzureRM.StreamAnalytics | Az.StreamAnalytics |
+| AzureRM.Subscription | Az.Subscription* |
+| AzureRM.Tags | Az.Tags* |
+| AzureRM.TrafficManager | Az.TrafficManager* |
+| AzureRM.UsageAggregates | Az.UsageAggregates |
+| AzureRM.Websites | Az.Websites |
+
+
+  ```*``` Module Name may change for 1.0
+
+  ```**``` Module not yet available
+
+  ```***``` Module may not appear in 1.0
+
+
+### Installing Az and AzureRM Side-by-Side - Discouraged
+
+It is discouraged for AzureRM and Az to be installed on the same machine, but there may be limited circumstances where this is needed for certain older scripts.
```

Comment:
@markcowl we should probably call out the need for `-AllowClobber` when installing the second module since there are some cmdlets in `AzureRM.Profile` that have the `Az` noun prefix